test/unit/tracks/text-track-controls.test.js

```diff
@@ -435,7 +436,11 @@ test('chapters should be displayed when remote track added and load event fired'
 
   equal(chaptersEl.track.cues.length, 2);
 
-  TestHelpers.triggerDomEvent(chaptersEl, 'load');
+  if (player.tech_.featuresNativeTextTracks) {
+    TestHelpers.triggerDomEvent(chaptersEl, 'load');
```

Comment:
So should this check for IE8 instead? or maybe just add a comment to specify that this is a protection for IE8?